### DISCUSSION TITLE
Improve UX for message handling

### DISCUSSION THREAD

**Discussion by marbetschar** [elementary/mail#609] [Ideas]
The current UX for message handling within the same conversation confuses me due to the following two reasons:

## Problem 1: Messages are rendered in the opposite direction to conversations

When I want to see the newest conversation I need to scroll upwards, but its the opposite for the newest message within a conversation.

## Problem 2: If I reply, the composer is added below the message it quotes

When I scroll upwards from the composer, I first see the old text content which is part of the cited message - that's quite confusion IMHO.

## Potential Solutions

For **problem 1** I think it would be best to render the newest message at the top - then we have the same "directions in time" as we use in the conversation list. Something like this (the timestamps in the screenshot are wrong - but you get the idea):

![new](https://user-images.githubusercontent.com/392542/125450698-2867647b-e49f-4a7a-8746-8d5ed40e9ca0.png)

For **problem 2** its probably best to replace the cited message with the composer entirely. This way, the user does not see the same content twice.

**Comment by janxkoci** [comment]
The order of messages in both panes is the same as in Gmail. It never confused me there and it doesn't confuse me here either. It works exactly to my expectations.
